ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh

```diff
@@ -4,7 +4,7 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-export LOKI_VERSION="2.2.1"
+export LOKI_VERSION="2.4.1"
```

Comment:
This one updates both loki and promtail in this file, wdyt @vrutkovs ?

Reply:
I'm fine as long as functionality is not lost, `ipi-install-loki` is most commonly used, so its safe enough